acm/odh-core/acm-observability/grafana-dashboard-edge-inference-health.yaml

```diff
@@ -0,0 +1,231 @@
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  annotations:
+    observability.open-cluster-management.io/dashboard-folder: Edge
+  labels:
+    grafana-custom-dashboard: "true"
+  name: grafana-dashboard-edge-inference-health
+data:
+  edge-inference-health.json: |-
```

Comment:
I guess the contents of this is what's in `acm/odh-core/acm-observability/files/grafana-edge-inference-health.json`, right? If so, then I think it would be cleaner to use Kustomize's [configMapGenerator](https://kubectl.docs.kubernetes.io/references/kustomize/kustomization/configmapgenerator/) functionality to put those two pieces together automatically, rather than having to do it manually whenever the dashboard gets changed). You'll see that there's a `secretGenerator` already in the `kustomization.yaml` which does something similar. WDYT?